docs/basics/download_nwb.ipynb

```diff
@@ -51,7 +51,7 @@
    "metadata": {},
```

Comment:
<p>Maybe rerun this and push without errors.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>